<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="btn_sell">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>130</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>販售商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_money">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>390</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>儲值</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_logout">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>390</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>登出</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buy">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>購買商品</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_money">
     <widget class="QPushButton" name="btn_back_m">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>60</pointsize>
       </font>
      </property>
      <property name="text">
       <string>儲值</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>我的金錢：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_money">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>請輸入金額</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>儲值金額：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_check_m">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>430</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>確定</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_show_money">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>240</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_sell">
     <widget class="QPushButton" name="btn_back_s">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_commodity">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>請輸入商品名稱</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_commodity_price">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>360</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>請輸入商品價格</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品販賣</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_count">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>360</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>請輸入商品數量</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_check_s">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>450</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>確定</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_buy">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <widget class="QPushButton" name="btn_back_b">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_commodity">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_count">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>數量</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_price">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>40</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>價格</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_back">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>470</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>上一頁</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_next">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>470</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>下一頁</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buy1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>購買</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buy2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>購買</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buy3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>購買</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_com1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_com2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_com3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>370</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_cnt2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_cnt1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_cnt3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>370</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_pr2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>260</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_pr1">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>140</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_pr3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>370</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_page">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>470</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>目前頁面：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_money">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>520</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>我的金錢：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_commodity">
     <widget class="QLabel" name="label_buy_title">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品購買</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_check_s_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>450</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>確定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buypage_back">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_commodity_price_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>360</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>請輸入商品購買數量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_total">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>360</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_com">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>名稱：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cnt">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>230</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>數量：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cpr">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>230</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>價格：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_money_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>470</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>我的金錢：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
